src/events.ts

```diff
@@ -141,25 +141,33 @@ export interface EventSource {
   metadata?: Record<string, any>;
 }
 
+/**
+ * Represents seeking position.
+ */
+export interface SeekPosition {
+  /**
+   * The relevant `Source`.
+   */
+  source: EventSource;
+  /**
+   * The position within the `source` in seconds.
+   */
+  time: number;
+}
+
 /**
  * Emitted when the player is about to seek to a new position.
  * Only applies to VoD streams.
```

Comment:
NIT: we could cross reference `timeShift` here for live streams.

Reply:
What do you mean?

Reply:
That's a great suggestion. Please see 5681080.
By using `@link` it will have them linked in the description for easy jumping.
See it in action here: https://cdn.bitmovin.com/player/reactnative/0.12.0/docs/interfaces/SeekEvent.html